scss/grouparoo.scss

```diff
@@ -168,6 +214,41 @@ header {
   }
 }
 
+.pipes {
+  // This pipes image has been manually set to 40% transparency
+  background: url("../public/images/home/pipes.png"),
+    linear-gradient(
+      225.01deg,
+      #000000 0%,
+      #0b2e4e 0.01%,
+      #0a2540 99.99%,
+      #06182c 100%
+    );
+  background-size: 450px;
```

Comment:
```suggestion
  background-repeat: repeat, no-repeat;
  background-size: 450px, auto;```

Reply:
Eliminates the vertical seams in the background.  The horizontal seam between the nav/header would need a new Layout file, but may not be a priority at this time.

Reply:
Good stuff! Will make this change.